src/serve/proxy.rs

```diff
@@ -91,6 +91,7 @@ impl ProxyBuilder {
                     Default::default()
                 }
             );
+            tracing::debug!("Proxy options: {:?}", opts); // Optional debug log
```

Comment:
I'd appreciate the comment to be removed, as it's obvious:

```suggestion
            tracing::debug!("Proxy options: {opts:#?}");
```